### PR TITLE
Events content on Find service start page

### PR DESCRIPTION
**Pull request by Kateharries**

### CODE REVIEW

File: app/views/results/_promoted-link.njk [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <div class="app-promoted-link">
-  <a class="govuk-link" href="https://getintoteaching.education.gov.uk/events">Talk to teacher training providers at an event near you</a>.
+  <a class="govuk-link" href="https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events?utm_source=find-postgraduate-teacher-training.service.gov.uk&utm_medium=referral&utm_campaign=find_banner&utm_content=Spring23events">Talk to teacher training providers at an event near you</a>.
```
Comment (simonwhatley) [inline app/views/results/_promoted-link.njk:2]: No need to add Google tags to the prototype code.

```suggestion
  <a class="govuk-link" href="https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events?utm_source=find-postgraduate-teacher-training.service.gov.uk">Talk to teacher training providers at an event near you</a>.
```

File: app/views/search/index.njk [hunk 2]
```diff
@@ -111,6 +111,15 @@
...
   </div>
+
+      <p class="govuk-body"><a class="govuk-link" href="https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events?utm_source=find-postgraduate-teacher-training.service.gov.uk&utm_medium=referral&utm_campaign=find_start&utm_content=Spring23events">Meet local teacher training providers at a Get Into teaching event</a> where you can: </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>get advice on your application</li>
+        <li>talk to existing teachers</li>
+        <li>connect with people like you who are interested in teaching</li>
+        </p>
+
+
```
Comment (simonwhatley) [inline app/views/search/index.njk:122]: I think you need to separate the content from the form. For example:

```
## Meet local teacher training providers

Meet local teacher training providers at a [Get Into teaching event](url) where you can:

- get advice on your application
- talk to existing teachers
- connect with people like you who are interested in teaching
```

**Review by simonwhatley — CHANGES_REQUESTED** [review]

File: app/views/search/index.njk [hunk 3]
```diff
@@ -111,6 +111,15 @@
 
     </div>
   </div>
+
+      <p class="govuk-body"><a class="govuk-link" href="https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events?utm_source=find-postgraduate-teacher-training.service.gov.uk&utm_medium=referral&utm_campaign=find_start&utm_content=Spring23events">Meet local teacher training providers at a Get Into teaching event</a> where you can: </p>
```
Comment (simonwhatley) [inline app/views/search/index.njk:115]: No need to add Google tags to the prototype code

```suggestion
      <p class="govuk-body"><a class="govuk-link" href="https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events?utm_source=find-postgraduate-teacher-training.service.gov.uk">Meet local teacher training providers at a Get Into teaching event</a> where you can: </p>
```

File: app/views/search/index.njk [hunk 4]
```diff
@@ -111,6 +111,15 @@
...
+      <ul class="govuk-list govuk-list--bullet">
+        <li>get advice on your application</li>
+        <li>talk to existing teachers</li>
+        <li>connect with people like you who are interested in teaching</li>
+        </p>
```
Comment (simonwhatley) [inline app/views/search/index.njk:120]:
```suggestion
      </ul>
```

**Comment by Kateharries** [comment]
[Add events content to first page on Find](https://trello.com/c/OwKID0wI/1030-add-events-content-to-first-page-on-find)

**Review by simonwhatley — APPROVED** [review]

**Pull request merged**